_data/harnesses/connect-sink-timestamp/ksql.yml

```diff
@@ -0,0 +1,129 @@
+answer:
+  steps:
+    - title:
+      content:
+        - action: skip
+          render:
+            file: tutorials/connect-sink-timestamp/ksql/markup/answer/short-answer.adoc
+
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/connect-sink-timestamp/ksql/markup/dev/init.adoc
+
+    - title: Get Confluent Platform
```

Comment:
Should we expand this title as well, since it's more than just Confluent Platform?

Reply:
I made it more general, "Set up environment"